.storybook/main.js

```diff
@@ -27,20 +27,18 @@ module.exports = {
     "@storybook/addon-cssresources",
   ],
   webpackFinal: async config => {
-    config.module.rules.push(
-      {
-        resolve: {
-          alias: {
-            storybook: path.resolve("./.storybook/"),
-          },
-        },
-      },
+    config.module.rules = [
+      ...config.module.rules,
       {
         test: /\.scss$/,
         loaders: ["style-loader", "css-loader", "sass-loader"],
         include: path.resolve(__dirname, "../"),
-      }
-    );
+      },
+    ];
+    config.resolve.alias = {
```

Comment:
It used to be on `module.resolve` 
https://github.com/acl-services/paprika/blob/962fb86bceeab241050906eb626028fdf8116a62/.storybook/webpack.config.js#L21-L25

and then got moved to `module.rules` in a previous PR
https://github.com/acl-services/paprika/commit/fca4da2562d4fbad7ca89b12b8bc60a3b6f8263d

@alexzherdev pointed it out to me, so I snuck a quick fix into the PR I was working on at the time.

Reply:
ok, interesting that the `resolve` block actually worked under `rules` or perhaps it didn't? 🤷‍♂️

Reply:
It did work, because of some Webpack magic.  But Alex didn't think we _should_.  There was a whole Slack thread about it, and I had him take a look at this part of the PR already in fact.

Reply:
Ok, I don't see that discussion, Perhaps you can reference that slack thread or copy in the discussion here to help the review process

Reply:
It was a DM thread... and I'm not always to keen to share Slack links in Paprika, since it's open source.